en/snap/platform/overview.md

```diff
@@ -193,7 +193,18 @@ On **Sandbox** mode:
 Done! Now your Shopify online shop is ready to use Midtrans as payment gateway.
 ![shopify](./../../../asset/image/shopify-17.png ':size=400')
 
-<hr><br><br>
+<hr>
+
+### Unable to Install Specific Payment Methods
```

Comment:
Since this should still be within the `Specific Payment Methods` section, use h4 instead of h3

Reply:
you right